autoprofile.py

```diff
@@ -33,15 +35,91 @@ def __init__(self):
         self.name = _("Automatic Profile")
         self.bio_enabled = False
         self.name_enabled = False
+        self.pfp_enabled = False
+        self.pfp_remove_latest = None
         self.raw_bio = None
         self.raw_name = None
+        self.pfp_degree = 0
 
     async def client_ready(self, client, db):
         self.client = client
 
+    async def autopfpcmd(self, message):
+        """Rotates your profile picture every n seconds with x degrees, usage:
+           (prefix)autopfp <timeout> <degrees> <remove previous (last pfp)>
+
+           Timeout - seconds
+           Degrees - 60, -10, etc
+           Remove last pfp - true or false, boolean, case insensitive"""
+
+        msg = utils.get_args(message)
+        if len(msg) != 3:
+            return await message.edit(_("<b>Autopfp requires three args. See the help for syntax.</b>"))
+
+        try:
+            timeout_autopfp = int(msg[0])
+        except ValueError as e:
+            logger.warning(str(e))
+            return await message.edit(_("<b>Wrong time.</b>"))
+
+        try:
+            degrees = int(msg[1])
+        except ValueError as e:
+            logger.warning(str(e))
+            return await message.edit(_("<b>Wrong degrees value.</b>"))
+
+        if msg[2].lower() == 'true':
```

Comment:
Use ast.literal_eval and check truthiness

Reply:
Make sure to catch valueerror and syntaxerror

Reply:
seems useless because it doesn't even work with 

> Use ast.literal_eval and check truthiness

seems useless because it doesn't work even with `true`/`false`

Reply:
It is more pythonic tho...